src/main/java/org/toradocu/translator/CommentTranslator.java

```diff
@@ -111,6 +111,21 @@ public static ThrowsSpecification translate(ThrowsTag tag, DocumentedExecutable
     return specs;
   }
 
+  /**
+   * If randoop sepcs are enabled and receiver name conflicts with one parameter name, adjust the
+   * receiver name.
+   *
+   * @param receiver original receiver name
+   * @param paramNames list of parameters names
+   * @return adjusted receiver name
+   */
+  private static String adjustReceiverName(String receiver, List<String> paramNames) {
```

Comment:
To simplify the code, can't we mangle the receiver name with the "-class-ID" suffix every time? Why do we need to check for a conflict?

Also, why do we check whether Randoop specifications are enabled? I think we should just modify the receiver name to include a fixed suffix.

Reply:
Everything you say is true, just remember we would have to change all the current goal file - am I wrong?

Reply:
No, I think you are right. I was thinking that a smart find and replace would be sufficient. Do you think is harder?

Reply:
Could be sufficient unless we have already some conflicts without knowing, such as parameters that are called `target`, I think (I didn't have time for these checks so this pull request was the fastest solution to proceed with the experiments)

Reply:
Could be harder if we have already conflicting names without knowing (such as parameters called `target` and involved in our translations), at least I think. I didn't have time for these checks so this fix was the fastest one to proceed with the experiments